userProfile/__tests__/functionality_unit.test.mjs

```diff
@@ -0,0 +1,57 @@
+import {findUserById} from '../userProfile.mjs'
+import {validateID} from '../../utilities/shared.mjs'
+
+// These test the pieces of functionality in the route that make it work.
+describe('userProfile endpoint functionality unit test (just testing helper functions). #functions_unit', () => {
+
+  it('No TPEN3 user ID provided.  User ID validation must be false.', () => {
+    expect(validateID()).toBe(false)
+  })
+  
+  it('Throws error and handles if id is 111', async () => {
+    try {
```

Comment:
It is a little wonky to do this try-catch in tests, though it is logically correct.  Below is a suggested code simplification. 

```
const res = await findUserById(111).catch(err => err)
expect(res.message).toBe('Internal Server Error')
```

Reply:
Really agree with that , corrected and updated.